web/src/SidebarResources.tsx

```diff
@@ -150,6 +178,14 @@ export class SidebarResources extends React.Component<SidebarProps> {
         ? "isOverview"
         : ""
 
+    if (listItems.length === 0 && options.resourceNameFilter) {
```

Comment:
hmm arguably if you click "tests only" and there are no tests, you should get this message then too? though i'm not actually sure we offer you the "tests only" button in that case... (you don't actually need to change this, just thinking aloud)

Reply:
yeah, changed the second condition to `this.props.items.length() !== 0` - it seems like in general if there were originally items and we filtered them all out, it's good to say no matches